bn_mp_grow.c

```diff
@@ -12,7 +12,7 @@ int mp_grow(mp_int *a, int size)
    /* if the alloc size is smaller alloc more ram */
    if (a->alloc < size) {
       /* ensure there are always at least MP_PREC digits extra on top */
-      size += (MP_PREC * 2) - (size % MP_PREC);
+      size += ((MP_PREC * 2) - 1) - ((size + (MP_PREC - 1)) % MP_PREC);
```

Comment:
Is the "extra" really necessary?
I would prefer that LTM does what I tell it and not something different, as benign as it might be.
That "extra" is also not documented, so if somebody uses that information to reduce the memory they cannot blame LTM if it fails.

Reply:
I agree with @czurnieden. I would **strongly** advise to not use any extra. This will ensure that our valgrind tests tell if something is going wrong.

Reply:
Adapted now not to use extra's, but simply round up to a multiple of MP_PREC. Build started, let's see how it goe's